src/main/kotlin/com/geospock/pubvote/people/People.kt

```diff
@@ -8,17 +8,16 @@ enum class People(val slackHandle: String, val role: Role = Role.PEOPLE_EARTHLIN
     KAI("kai", Role.MANAGEMENT),
     JON("jon", Role.MANAGEMENT),
     DAVID_W("david", Role.MANAGEMENT),
-    JAMES_G("james"),
+    JAMES("james", Role.MANAGEMENT),
     BOB("bob"),
     BENITA("benita", Role.EXECUTIVE_LEADERSHIP_TEAM),
     DAN("dans"),
     XAVI("xavi"),
     SAM_S("sam"),
     FELIX_M("felix"),
-    HUW("huw", Role.EXECUTIVE_LEADERSHIP_TEAM),
+    HUW("huw"),
     NICOLA("nicola"),
-    DOM("dom"),
```

Comment:
QQ

Reply:
Sorry man, it's not personal... mostly 😛